sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -197,8 +197,16 @@ def rank_error(msg):
           # Skip empty messages.
           if m.messageImportance is None:
             continue
-          _LOGGER.info(message)
-          if str(m.messageImportance) == 'JOB_MESSAGE_ERROR':
+          message_importance = str(m.messageImportance)
```

Comment:
should we have catch-all clause logged as INFO?

Reply:
e.g. for  JOB_MESSAGE_IMPORTANCE_UNKNOWN

Reply:
That's reasonable, done

Reply:
Also catches any new enums that may be added (doesn't seem likely but at the very lease we'll be logging that output)